bcda/client/bluebutton.go

```diff
@@ -133,8 +133,6 @@ func NewBlueButtonClient(config BlueButtonConfig) (*BlueButtonClient, error) {
 	return &BlueButtonClient{client, maxTries, retryInterval, config.BBServer, config.BBBasePath}, nil
 }
 
-type BeneDataFunc func(string, string, string, string, time.Time) (*models.Bundle, error)
```

Comment:
This type was remove because we no longer have the same function signature for all resource types (EOB has serviceDate parameter).

When we were using `BeneDataFunc` we introduced another interface (all BB funcs had the same signature) that had to be satisfied. Since serviceDate does not make sense w.r.t to Coverage and Patient types, it didn't make sense to update `BeneDataFunc` to include ServiceDate filter.